src/utilities/elevation/Elevation.props.ts

```diff
@@ -0,0 +1,19 @@
+import { TElevationLevel } from './Elevation.types';
+
+type PElevation = {
+    /**
+     * Elevation refers to the z-index depth that an element sits on.
+     * The base Elevation is 0 and has no depth.
+     * Subsequent layers increment this Elevation value.
+     * In total, there are 6 Elevation levels, not including the base level 0.
+     * @default 0
+     * */
+    elevation?: TElevationLevel;
+    /**
+     * If a hover-state should require a higher (or lower) elevation
+     * @default 0
```

Comment:
nit: this was a mistake on my end as well ... the default for this is `elevation` ... not `0`, but since most of the time `elevation` will be `0`, it is kinda true. Just to make sure we should add `@default elevation` here